src/main/java/seedu/address/model/order/CollectionTypeContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.order;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
```

Comment:
Update the comments here to "Order" and "CollectionType"?

Reply:
Great spot!